libs/typescript/ts-utils/common-utils.ts

```diff
@@ -148,13 +148,37 @@ export function assertNotNull<T>(
   return value;
 }
 
+export function checkIfNull(value: string | null | undefined, msg: string) {
+  if (
+    value === null ||
+    value === undefined ||
+    (typeof value === 'string' && !value.length)
+  ) {
+    console.log(msg);
+    return true;
+  }
+  return false;
+}
+
 export function getEnvString(varName: string) {
   return assertNotNull(
     process.env[varName],
     `Env variable '${varName}' is missing.`,
   );
 }
 
+export function getSecretEnvString(varName: string) {
+  if (
+    checkIfNull(process.env[varName], `Env variable '${varName}' is missing.`)
+  ) {
+    return '';
+  } else {
+    let path = process.env[varName];
+    const rootDir = getEnvString('GIT_ROOT');
```

Comment:
Is this the only place we use `rootDir`. If no, we can move it somewhere top-level and export and reuse it.

Reply:
We were using another function elsewhere that I removed and used `rootDir` in its place. https://github.com/metacraft-labs/DendrETH/pull/373/commits/8d97bb2fac4363dba9e24b781b7499c6fcf9aca0